src/components/PatchesPage/PatchCard.tsx

```diff
@@ -49,10 +49,34 @@ export const PatchCard: React.VFC<Props> = ({
 }) => {
   const createDate = new Date(createTime);
   const getDateCopy = useDateFormat();
-  const { taskStatusStats, id: versionId } = versionFull || {};
+  const { taskStatusStats, id: versionId, projectMetadata } = versionFull || {};
   const { stats } = groupStatusesByUmbrellaStatus(
     taskStatusStats?.counts ?? []
   );
+
+  let patchProject = null;
+  if (pageType === "project") {
+    patchProject = (
+      <StyledRouterLink
+        to={getUserPatchesRoute(author)}
+        data-cy="user-patches-link"
+      >
+        <strong>{authorDisplayName}</strong>
+      </StyledRouterLink>
+    );
+  } else {
+    patchProject = projectIdentifier ? (
+      <StyledRouterLink
+        to={getProjectPatchesRoute(projectIdentifier)}
+        data-cy="project-patches-link"
+      >
+        <strong>{projectIdentifier}</strong>
+      </StyledRouterLink>
+    ) : (
+      `${projectMetadata.owner}/${projectMetadata.repo}`
+    );
```

Comment:
Is the reasoning behind this that there is no link we can take the user too to see the rest of the patches?

Reply:
Yep -- we don't currently have an equivalent repo query for a project's [Patches](https://github.com/evergreen-ci/evergreen/blob/74886a90a000ab5637663906a563a1ef60d236f5/graphql/project_resolver.go#L28-L52) resolver that could return all its patches. I think it'd be worth adding though! EVG-18197